docs/extended-tour.md

```diff
@@ -172,6 +173,40 @@ User of your module runs 'npm install'
 
 This binary file `../lib/binding/module.node` is what `require()` points to within Node.js.
 
+# Clang Tools
+
+This skeleton uses two clang llvm tools for automated formatting and static fixes. 
```

Comment:
Ideally say `clang/llvm` to denote the projects are the same (just different parts: `llvm` is the term for the entire project and `clang` is the term for the "front-end" sub-projects).